utils.go

```diff
@@ -211,18 +211,30 @@ func getRootlessStorageOpts(rootlessUID int, systemOpts StoreOptions) (StoreOpti
 	} else {
 		opts.GraphRoot = filepath.Join(dataDir, "containers", "storage")
 	}
+	opts.GraphDriverName = os.Getenv("STORAGE_DRIVER")
 	if path, err := exec.LookPath("fuse-overlayfs"); err == nil {
-		opts.GraphDriverName = "overlay"
-		opts.GraphDriverOptions = []string{fmt.Sprintf("overlay.mount_program=%s", path)}
-		for _, o := range systemOpts.GraphDriverOptions {
-			if strings.Contains(o, "ignore_chown_errors") {
-				opts.GraphDriverOptions = append(opts.GraphDriverOptions, o)
-				break
+		if opts.GraphDriverName == "" {
```

Comment:
do we need to check at all for `fuse-overlayfs` if the graphdriver is set and has a value different than "overlay"?

Reply:
No, so I cleaned this checking up.